.gitignore

```diff
@@ -16,4 +16,7 @@ coverage
 # VSCode
 .vscode
 .tmp
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Idea
+**/.idea/*
```

Comment:
What is easy to configure it on idea IDE?
You may prepare a developer installation guide, if your memory is fresh enough.